src/pages/Help/Overview/index.jsx

```diff
@@ -68,7 +68,7 @@ const Overview = () => {
           </div>
           <div className="col-md-6 p-md-0">
             <h2 className="text-primary-color font-size-sub-heading">
-              FOSSology's Graphical User Interface
+              FOSSology Graphical User Interface
```

Comment:
Can you please explain the change? Existing sentence looks correct to me.

Reply:
Its Fossology`s and the correct sentence is Fossology